src/project/signing.rs

```diff
@@ -25,7 +25,11 @@ impl Project {
 
     /// Get the name for .bisign files
     pub fn get_sig_name(&self, pbo: &str) -> Result<String, HEMTTError> {
```

Comment:
This should have a check for no prefix instead of checking if the prefix is the pbo name

Reply:
good point will change that